phoenicis-tools/src/main/java/org/phoenicis/tools/lnk/LnkParser.java

```diff
@@ -0,0 +1,156 @@
+package org.phoenicis.tools.lnk;
+
+import org.apache.commons.io.IOUtils;
+import org.phoenicis.configuration.security.Safe;
+import org.phoenicis.tools.files.FilesManipulator;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+
+/**
+ * This file parses .lnk files
+ * https://msdn.microsoft.com/en-us/library/dd871305.aspx
+ */
+@Safe
+public class LnkParser extends FilesManipulator {
```

Comment:
For me the distribution of responsibility between `LinkParser`, `LinkFile` and `LinkData` is not quite clear.

Reply:
LnkData corresponds to a part of the .lnk structure (You can read the spec for more details is you are curious about it).
LnkFile is just a data transfert object, 
LnkParser is the actual stateless parser that returns a LnkFile